client/src/LayoutComponents/ProjectDetails/ProjectDetails.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import './ProjectDetails.css';
+import github from '../../images/github.svg';
+import pc from '../../images/pc.svg';
+import ProjectView from '../../SharedComponents/ProjectView';
+import StudentCard from '../../SharedComponents/StudentCard';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+class ProjectDetails extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      projects: [],
+    };
+  }
+  componentDidMount() {
+    const { id } = this.props;
+    // id coming from cohorts and studentProfile pages
+    axios
+      .get(`api/project/${id}`)
+      .then((res) => {
+        const resultArray = res.data.map((e) => e);
+        this.setState({
+          projects: resultArray,
+        });
+      })
+      .catch((error) => error.response);
+  }
+  render() {
+    const { projects } = this.state;
+
+    return (
+      <div className='projectdetails-container'>
+        {!projects.length ? (
+          <div className='loading-container'>
+            <h1 className='loading'> </h1>
+          </div>
+        ) : (
+          <div>
+            <section className='projects-details'>
```

Comment:
@kholoudfann If you want, this could show all projects instead of just the first project:
```
<section className='projects-details'>
  {projects.map((project) => (
    <div className='projects-info' key={project.id}>
      <h2 className='projects-title>{project.title}</h2>
      <p className='project-desc'>{project.description}</p>
       ....
    </div>
  )}
</section>
```

Reply:
Thank you for all your reviewers , but in this page  we want the projects that team work in it , not all projects .